package.json

```diff
@@ -18,8 +18,9 @@
     "bazel:buildifier": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,output-group,package-name,package-on-top,redefined-variable,repository-name,same-origin-load,string-iteration,unused-variable,unsorted-dict-items,out-of-order-load",
     "bazel:format-lint": "yarn -s bazel:buildifier --lint=warn --mode=check",
     "dev-app": "ibazel run //src/dev-app:devserver",
-    "test": "bazel test //src/... --test_tag_filters=-e2e,-browser:firefox-local --build_tag_filters=-browser:firefox-local --build_tests_only",
-    "test-firefox": "bazel test //src/... --test_tag_filters=-e2e,-browser:chromium-local --build_tag_filters=-browser:chromium-local --build_tests_only",
+    "test": "node ./scripts/run-component-tests.js",
+    "test-local": "yarn -s test --local",
```

Comment:
I'm not sure what `local` refers to in this context. Is it just to indicate that we shouldn't be running this script on the CI?

Reply:
We decided in another PR that we want to align the old `gulp:static` stuff with "local" that aligns with what is used internally in Google. I didn't feel strongly about it.